src/vl-map-search.js

```diff
@@ -146,22 +174,40 @@ export class VlMapSearch extends vlElement(HTMLElement) {
       this.__choiceEventListenerRegistered = true;
       this.__choiceEventListener = this._selectElement.addEventListener('choice', (event) => {
         if (event && event.detail && event.detail.choice) {
-          fetch(this.locationUrl + event.detail.choice.value).then((response) => {
-            return response.json();
-          }).then((data) => {
-            if (data && data.LocationResult) {
-              if (this._onSelect) {
-                this._onSelect(data);
-              } else if (this._map) {
-                this._map.zoomTo([
-                  data.LocationResult[0].BoundingBox.LowerLeft.X_Lambert72,
-                  data.LocationResult[0].BoundingBox.LowerLeft.Y_Lambert72,
-                  data.LocationResult[0].BoundingBox.UpperRight.X_Lambert72,
-                  data.LocationResult[0].BoundingBox.UpperRight.Y_Lambert72,
-                ]);
+          const value = event.detail.choice.value;
+
+          // TODO stefanborghys: 28/01/21 prefer to use instanceof but could not import LambertCoordinaat in the test and replaced it for now
+          // instanceof LambertCoordinaat
+          if (value.x !== undefined && value.y !== undefined ) {
+            this._map.zoomToGeometry({
+              type: 'Point',
+              coordinates: [value.x, value.y],
+            }, 10);
+          } else if (value instanceof Object) {
+            this._map.zoomTo([
+              value.BoundingBox.LowerLeft.X_Lambert72,
+              value.BoundingBox.LowerLeft.Y_Lambert72,
+              value.BoundingBox.UpperRight.X_Lambert72,
+              value.BoundingBox.UpperRight.Y_Lambert72,
+            ], 14);
```

Comment:
Is er een onderscheid (nodig) tussen zoomToGeometry en zoomTo?
Die zoomTo lijkt mij toch ook gewoon ne zoomToGeometry(een_rechthoek)?

Reply:
dacht van niet, maar kan mis zijn. Had daarvoor gekeken in de vl-mapactions-custom-map.js van de vl-mapcations en daar gaat men ook een zoomToExtent() en zoomToGeometry() uit elkaar trekken en apart aanbieden. Er gebeurd daar een nog iets andere verwerking op als ik het nalees.